donkeycar/parts/keras_resnet18.py

```diff
@@ -0,0 +1,191 @@
+import tensorflow as tf
+from tensorflow.keras.layers import ZeroPadding2D, Input, GlobalAveragePooling2D,GlobalMaxPooling2D,Dense
+from tensorflow.keras.layers import Convolution2D,MaxPooling2D,BatchNormalization
+from tensorflow.keras.layers import Activation,Dropout,Flatten
+from tensorflow.keras.models import Model,Sequential
+from keras_applications.imagenet_utils import _obtain_input_shape, get_submodules_from_kwargs
+import os
+import warnings
+
+# Those are mandatory for ResNet function to work   
+backend = tf.compat.v1.keras.backend
+layers = tf.keras.layers
+models = tf.keras.models
+utils = tf.keras.utils
+
+WEIGHTS_PATH = 'https://raw.githubusercontent.com/cl3m3nt/resnet/master/resnet18_cifar100_top.h5'
+WEIGHTS_PATH_NO_TOP = 'https://raw.githubusercontent.com/cl3m3nt/resnet/master/resnet18_cifar100_no_top.h5'
+
+
+def identity_block(input_tensor, kernel_size, filters, stage, block):
+    filters1, filters2 = filters
+    if backend.image_data_format() == 'channels_last':
```

Comment:
This following code seems to re-appear a couple of times, maybe factor this into a small helper function.

Reply:
Yes, same here I followed what keras-team did for resnet50.
There a 2 occurences of the same code, which could be acceptable don't you think ?